sigstore/sign.py

```diff
@@ -188,3 +212,61 @@ class SigningResult(BaseModel):
     """
     A record of the Rekor log entry for the signing operation.
     """
+
+    def _to_bundle(self) -> Bundle:
```

Comment:
This can wait until a follow-up, but the amount of round-tripping here (`str -> bytes`, base64 decoding, etc) suggests that the current `SigningResult` field types aren't ideal 🙂 

They're already part of the public API, however, so any internal changes to them will probably need backwards-compatible properties.

Reply:
+1 to the sentiment, but I understand the need for compatibility :)